modules/thunder_updater/src/Command/GenerateThunderUpdateCommand.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Drupal\thunder_updater\Command;
+
+use Drupal\Console\Command\Shared\ConfirmationTrait;
+use Drupal\Console\Command\Shared\ModuleTrait;
+use Drupal\Console\Extension\Manager;
+use Drupal\Console\Utils\Site;
+use Drupal\thunder_updater\Generator\ThunderUpdateGenerator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Console\Core\Command\Command;
+use Drupal\Console\Core\Style\DrupalStyle;
+
+/**
+ * Generate Thunder update command class.
+ *
+ * TODO: Add support for version option, where entry in checklist will be added.
+ *
+ * @Drupal\Console\Annotations\DrupalCommand (
```

Comment:
Is the annotation needed? Didn't found anything about that in the docs...
https://hechoendrupal.gitbooks.io/drupal-console/content/en/extending/creating-custom-commands.html

Reply:
I have used `generate:command` and it adds that annotation. I have also checked and without annotation translation doesn't work, so I'll leave it as it is.